File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        num_hash = {}
        for index, num in enumerate(nums):
            complement = target - num
            if complement in num_hash:
                return [num_hash[complement], index]
            
            num_hash[num] = index

File: Data Structures & Algorithms/concatenation-of-array/submission-0.go
func getConcatenation(nums []int) []int {
    n := len(nums)
    m := 2 * n

    newArr := []int{}

    for i := 0; i < m; i++ {
        index := i % n
        newArr = append(newArr, nums[index])
    }

    return newArr
}
